src/components/Onboarding/Screens/Apps/IssuanceSettings/IssuanceChart.js

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo } from 'react'
+import { ChartBase } from '@components/Onboarding/kit/ChartComponents'
+import { ResponsiveLine } from '@nivo/line'
+import { useCharts } from '@/providers/Charts'
+import { generateElements } from '@/utils/conviction-modelling-helpers'
+
+const DEFAULT_MAX_MONTH = 12
+const DEFAULT_INCREMENT = 12 / 365
+
+const calculateRatio = (
+  initialRatio,
+  targetRatio = 0.001,
+  maxAdjustmentRatioPerYear
+) => {
+  let balance = initialRatio
+  let supply = 1
+  const ratio = [balance / supply]
+  for (let i = 0; i < 365; i++) {
+    const adj = ((1.0 - balance / supply / targetRatio) / 365) * supply
+    const maxAdj = maxAdjustmentRatioPerYear / 365 / targetRatio
+    const totalAdj = Math.sign(adj) * Math.min(Math.abs(adj), maxAdj)
+    balance += totalAdj
```

Comment:
I don't understand why both `balance` and `supply` are the sum of `totalAdj` 🤔

@sembrestels would you mind clarifying this for me 🙏